addon/components/file-picker.js

```diff
@@ -211,39 +211,47 @@ export default Component.extend({
   },
   /* Drag'n'Drop events */
   dragOver: function(event) {
-    if (event.preventDefault) {
-      event.preventDefault();
+    if (this.get('dropzone') === true) {
+      if (event.preventDefault) {
```

Comment:
Actually I don't know what the default behavior would be. I thought it would be a good idea but you can correct me if I'm wrong.